components/dashboards-web-component/src/designer/components/WidgetConfigurationPanel.jsx

```diff
@@ -1,46 +1,106 @@
 /*
- *  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
  *
- *  WSO2 Inc. licenses this file to you under the Apache License,
- *  Version 2.0 (the "License"); you may not use this file except
- *  in compliance with the License.
- *  You may obtain a copy of the License at
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
  *
- *  http://www.apache.org/licenses/LICENSE-2.0
+ *     http://www.apache.org/licenses/LICENSE-2.0
  *
- *  Unless required by applicable law or agreed to in writing,
- *  software distributed under the License is distributed on an
- *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- *  KIND, either express or implied.  See the License for the
- *  specific language governing permissions and limitations
- *  under the License.
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
  *
+ * Intellij Copyright Template.txt
+ * Displaying Intellij Copyright Template.txt.
  */
 
 import React from 'react';
+import _ from 'lodash';
 
 import Drawer from 'material-ui/Drawer';
 import Checkbox from 'material-ui/Checkbox';
+import TextField from 'material-ui/TextField';
+import SelectField from 'material-ui/SelectField';
+import MenuItem from 'material-ui/MenuItem';
+
+import List, {ListItem} from 'material-ui/List';
+import Subheader from 'material-ui/Subheader';
+import Divider from 'material-ui/Divider';
+
 
 import {dashboardLayout} from '../../utils/WidgetLoadingComponent';
 import {pubsubComponent} from '../../utils/PubSubComponent';
 
-import {FormattedMessage} from 'react-intl';
+
+// import {FormattedMessage} from 'react-intl';
+import Dialog from 'material-ui/Dialog';
+import FlatButton from 'material-ui/FlatButton';
+
 
 const styles = {
     widgetDrawer: {
-        background: '#192830'
+        background: '#192830',
+        zIndex: 0
     }
 };
 
+const labelStyle = {
+    fontSize: 12
+};
+
+const stylePanelHeader = {
+    color: ' #d7dbdd ',
```

Comment:
Remove spaces

Reply:
I don't think that's a good idea the spaces are needed for clarity and readability

Reply:
They are unnecessary